pkgs/development/python-modules/csv2md/default.nix

```
@@ -0,0 +1,31 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, setuptools
+}:
+
+buildPythonPackage rec {
+  pname = "csv2md";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "lzakharov";
+    repo = "csv2md";
+    rev = "v${version}";
+    hash = "sha256-INwZRbyJmyDd5SV7PBdqUc/3FO91upTgaxBfP/Giaw4=";
+  };
+
+  propagatedBuildInputs = [
```

```suggestion
  build-system = [
```

I think `setuptools` should be in `build-system` or `nativeBuildInputs` (now deprecated)

---

How about `format = "setuptools";` and removing `propagatedBuildInputs` completely? Seems the cleanest but [manual](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md) doesn't mention it as possibility.